stylesheets/print.styl

```diff
@@ -0,0 +1,139 @@
+body 
+	width 100% !important
+	margin 0 !important
+	padding 0 !important
+	line-height 1.45
+	font-size 14pt
+*:not(.news-item-default-image):not(.date-flag):not(.icon):not(.cd-ic),
```

Comment:
Newlines before new selector blocks to make things clearer.

This `:not` chain could use a comment, too.